internal/runes/runes.go

```diff
@@ -1,22 +1,10 @@
 package runes
 
-import "github.com/samber/lo"
-
-// IndexNotByte is similar with strings.IndexByte but showing the opposite behavior.
-func IndexOfNot(x []rune, c byte) int {
-	i := lo.LastIndexOf(x, ' ')
-
-	if i == -1 && i+1 < len(x) {
-		return i + 1
-	}
-
-	return -1
-}
-
-// LastIndexNotByte is similar with strings.LastIndexByte but showing the opposite behavior.
-func LastIndexNotByte(s string, c byte) int {
-	for i := len(s) - 1; i >= 0; i-- {
-		if s[i] != c {
+// IndexOfNot returns the index of the first rune in x that is not c
+func IndexOfNot(x []rune, c rune) int {
```

Comment:
Well, I should not have reviewed commit-by-commit, because now it all makes more sense 🍭

Reply:
Yeah, sorry! I also like commit-by-commit PRs, but they are generally easier to create for features I know I have a clear path in mind. In the case of this bug here, it was a bit more messy.